projects/rero/ng-core/src/lib/record/record.handle-error.service.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2020-2023 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { Observable, throwError } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RecordHandleErrorService {
+  /**
+   * Constructor
+   * @param translateService - TranslateService
+   */
+  constructor(protected translateService: TranslateService) { }
+
+  /**
+   * handle http error
+   * @param error - HttpErrorResponse
+   * @returns Observable
+   */
+  handleError(error: HttpErrorResponse, resourceName?: string): Observable<never> {
```

Comment:
resourceName seems not used.

Reply:
This parameter is used in the UI. It's to have the same signature.